### PR TITLE
Don't bleed Stats api from management

### PR DESCRIPTION
**Pull request by AbfrmBlr**

### CODE REVIEW

File: management/management-registry/src/main/java/org/terracotta/management/registry/DelegatingSample.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terracotta.management.registry;
+
+import java.io.Serializable;
+
+public class DelegatingSample<T extends Serializable> implements Sample<T> {
```
Comment (mathieucarbou) [inline management/management-registry/src/main/java/org/terracotta/management/registry/DelegatingSample.java]: `serialVersionUID` is missing

**Comment by myronkscott** [comment]
Can we rebase and merge this?

File: lease/api/src/test/java/org/terracotta/lease/LeaseMaintenanceThreadTest.java [hunk 2]
```diff
@@ -57,7 +57,9 @@ public void callsRefreshOccasionally() throws Exception {
 
     verify(timeSource, timeout(3000L).times(1)).sleep(eq(2000L));
     verify(leaseMaintainer, times(1)).refreshLease();
-    
+
+    Thread.sleep(3000);
+
```
Comment (chrisdennis) [inline lease/api/src/test/java/org/terracotta/lease/LeaseMaintenanceThreadTest.java]: Pretty sure Myron fixed this test recently.

Comment (AbfrmBlr) [reply lease/api/src/test/java/org/terracotta/lease/LeaseMaintenanceThreadTest.java]: But its racy, the test thread sometimes is too eager to tick the time

Comment (chrisdennis) [reply lease/api/src/test/java/org/terracotta/lease/LeaseMaintenanceThreadTest.java]: What do you mean it's too eager? It just advances a counter.

**Pull request merged**